perf/perf.js

```diff
@@ -212,6 +226,43 @@ async function createIndex(bench, opts) {
   await rep.close();
 }
 
+/**
+ * @param {Bench} bench
+ * @param {number} i
+ */
+async function benchmarkWriteReadRoundTrip(bench, i) {
+  const rep = await makeRep();
+  const key = `k${i}`;
+  const value = i;
+  let {promise, resolve} = resolver();
+  rep.subscribe(tx => tx.get(key), {
+    onData(res) {
+      resolve(res);
+    },
+  });
+  // onData is called once when calling subscribe
+  await promise;
+
+  const write = rep.register('write', tx => tx.put(key, value));
+  bench.name = 'roundtrip write/subscribe/get';
+  bench.size = 1;
+  bench.formatter = formatOpPerSecond;
+  bench.reset();
+
+  // reset these.
+  ({promise, resolve} = resolver());
+
+  await write();
+
+  const res = await promise;
+  if (res !== value) {
+    throw new Error();
```

Comment:
Do you want to put a message in here?

Reply:
doesn't matter... it is more about making sure the code is not optimized away.